frontend/src/components/forms/Signup.tsx

```diff
@@ -158,21 +145,20 @@ const Signup = ({
                 Create Account
               </Button>
             </Box>
-          </Flex>
-          <Flex
-            top="80%"
-            left="6%"
-            width="100%"
+          
+            <Flex 
+            width="80%"
             direction="row"
-            position="absolute"
             alignContent="center"
-          >
-            <Text variant="loginSecondary" paddingRight="1.1%">
-              Already have an account?
-            </Text>
-            <Text variant="loginTertiary" onClick={onLogInClick}>
-              Log In Now
-            </Text>
+            marginTop="3%"
+            >
```

Comment:
So this can also be simplified to just use flexbox and not use `margin` and `padding`:
```suggestion
     <Box w="80%">
       <Flex gap="10px">
         <Text variant="loginSecondary">
            Already have an account?
         </Text>
         <Text variant="loginTertiary" onClick={onLogInClick}>
           Log In Now
         </Text>
       </Flex>
     </Box>
```